readthedocs/settings/base.py

```diff
@@ -906,6 +906,7 @@ def DOCKER_LIMITS(self):
 
     # MailerLite API for newsletter signups
     MAILERLITE_API_SUBSCRIBERS_URL = 'https://api.mailerlite.com/api/v2/subscribers'
+    MAILERLITE_API_ONBOARDING_GROUP_URL = None
```

Comment:
Are we going to use the other value somewhere else? Looks like we are using just one of these settings.

Reply:
Yea, it's just group specific so didn't get included in the PR.

Reply:
@ericholscher Do we need to create an -ops PR for this?

Reply:
https://github.com/readthedocs/readthedocs-ops/pull/1254